api/services/ocean/rightsizing/schemas/oceanRightsizingRuleRecommendationApplicationOverheadValues.yaml

```diff
@@ -0,0 +1,25 @@
+type: object
+title: Ocean Rightsizing Rule Recommendation Application Overhead Values
+description: >
+  Determines the Ocean Rightsizing rule recommendation application overhead values
+properties:
+  cpu:
+    type: object
+    properties:
+      lower:
```

Comment:
'lower' and 'upper' doesn't seem to be the valid values for both overhead values and applicationboudaries object. It should be 'min' and 'max'. Please check.